File: game-server/src/main/java/work/fking/pangya/game/packet/handler/LoginBonusStatusPacketHandler.java
package work.fking.pangya.game.packet.handler;

import io.netty.buffer.ByteBuf;
import work.fking.pangya.game.net.ClientPacketHandler;
import work.fking.pangya.game.GameServer;
import work.fking.pangya.game.player.Player;
import work.fking.pangya.game.packet.outbound.LoginBonusReplyPacket;

public class LoginBonusStatusPacketHandler implements ClientPacketHandler {

    @Override
    public void handle(GameServer server, Player player, ByteBuf packet) {
        player.channel().writeAndFlush(new LoginBonusReplyPacket());
    }
}

File: game-server/src/main/kotlin/work/fking/pangya/game/packet/handler/LoginBonusStatusPacketHandler.kt
package work.fking.pangya.game.packet.handler

import io.netty.buffer.ByteBuf
import work.fking.pangya.game.GameServer
import work.fking.pangya.game.net.ClientPacketHandler
import work.fking.pangya.game.packet.outbound.LoginBonusReplyPacket
import work.fking.pangya.game.player.Player

class LoginBonusStatusPacketHandler : ClientPacketHandler {
    override fun handle(server: GameServer, player: Player, packet: ByteBuf) {
        player.channel().writeAndFlush(LoginBonusReplyPacket())
    }
}
